lms/djangoapps/instructor/access.py

```diff
@@ -33,31 +33,15 @@
 }
 
 
-def list_with_level(course, level):
+def list_with_level(course_id, level):
     """
     List users who have 'level' access.
 
     `level` is in ['instructor', 'staff', 'beta'] for standard courses.
     There could be other levels specific to the course.
     If there is no Group for that course-level, returns an empty list
     """
-    return ROLES[level](course.id).users_with_role()
-
-
-def list_with_level_from_course_key(course_key, level):
```

Comment:
I am removing the duplicated function that used the `course_key` and then converting the original function to use a course-run key. We were passing in the entire `course` but the original function only needs the course-id/course-run key.